nflow-jetty/src/main/java/io/nflow/jetty/mapper/WebApplicationExceptionMapper.java

```diff
@@ -0,0 +1,16 @@
+package io.nflow.jetty.mapper;
+
+import static javax.ws.rs.core.Response.status;
+
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+public class WebApplicationExceptionMapper implements ExceptionMapper<WebApplicationException> {
+  @Override
+  public Response toResponse(WebApplicationException e) {
+    return status(e.getResponse().getStatus()).entity(e.getMessage()).build();
```

Comment:
Should we return a json object instead of plain text string?
For example
 ```json
{ "error": "<message>" }
```

Reply:
Yes, fixed.